ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -159,23 +159,22 @@ objects:
         trap 'jobs -p | xargs -r kill || true; exit 0' TERM
 
         function fips_check() {
-          oc --insecure-skip-tls-verify --request-timeout=60s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}' > /tmp/nodelist
+          oc --request-timeout=20s get nodes -o jsonpath --template '{range .items[*]}{.metadata.name}{"\n"}{end}' > /tmp/nodelist
           while IFS= read -r i; do
-            oc -n default --insecure-skip-tls-verify --request-timeout=60s debug --image centos:7 node/$i -- cat /proc/sys/crypto/fips_enabled > /tmp/enabled
+            out=$(oc debug node/"$i" -- cat /proc/sys/crypto/fips_enabled || true)
             if [[ "${CLUSTER_VARIANT}" =~ "fips" ]]; then
-              if [[ $(< /tmp/enabled) == "0" ]]; then
+              if [[ ! "${out}" =~ "1" ]]; then
                 echo fips not enabled in node "$i" but should be, exiting
                 exit 1
               fi
             else
-              if [[ $(< /tmp/enabled) == "1" ]]; then
+              if [[ ! "${out}" =~ "0" ]]; then
```

Comment:
why does this have to be a regex check?  Why not `-eq 0`?

Reply:
:shrug :eyes:

Reply:
fixed it, thanks